etcdadm/etcdadm

```diff
@@ -6,72 +6,70 @@ unset ETCDCTL_ENDPOINT
 
 export | grep ETCDCTL
 
+set -o nounset
+set -o errexit
+set -o pipefail
```

Comment:
this is dangerous, it makes commands like `cat BIGFILE |  grep -q .` to fail, as `cat` receives `SIGPIPE` when `grep` exits and therefore exits with non 0 code, causing whole expression to fail